src/App.css

```diff
@@ -37,6 +37,10 @@
   }
 }
 
-.modal-70w {
-  width: 70%;
+.modal {
+  width: 100%;
+}
+
+.blurb {
+  text-align: center;
 }
```

Comment:
Good work on starting this lab! 

- Your modal is great (this was lab I know:)) but we can revisit your modal styles to adjust the size (make it smaller) 👍